python/ray/autoscaler/vsphere/defaults.yaml

```diff
@@ -0,0 +1,123 @@
+# An unique identifier for the head node and workers of this cluster.
+cluster_name: default
+
+# The maximum number of workers nodes to launch in addition to the head
+# node.
+max_workers: 2
+
+# The autoscaler will scale up the cluster faster with higher upscaling speed.
+# E.g., if the task requires adding more nodes then autoscaler will gradually
+# scale up the cluster in chunks of upscaling_speed*currently_running_nodes.
+# This number should be > 0.
+upscaling_speed: 1.0
+
+# This executes all commands on all nodes in the docker container,
+# and opens all the necessary ports to support the Ray cluster.
+# Empty string means disabled.
+docker: {}
+
+# If a node is idle for this many minutes, it will be removed.
+idle_timeout_minutes: 5
+
+# Cloud-provider specific configuration.
+provider:
+    type: vsphere
+
+    vsphere_config:
+      credentials:
+        admin_user: <username>
+        admin_password: <password>
+        server: <server-ip>
+      datacenter: <datacenter>
```

Comment:
Similar concern here, what happens if the user forgets to provide any of these fields in their YAML? Can we make sure it has a reasonable default or has a good error message?

Reply:
Fixed with some default value.